Tests/MountebankSwiftTests/Api/MountebankIntegrationTests.swift

```diff
@@ -126,6 +126,36 @@ final class MountebankIntegrationTests: XCTestCase {
         XCTAssertEqual(response.stubs.count, 1)
     }
 
+    // Added test is encoding of all these types goes well
+    func testDiffentResponseBodyTypes() async throws {
+        let imposter = Imposter(
+            port: 1234,
+            networkProtocol: .http,
+            name: "Imposter with various body Types",
+            stubs: [
+                SampleFile.png,
+                SampleFile.jpg,
+                SampleFile.json,
+                SampleFile.pdf,
+                SampleFile.txt,
+                SampleFile.mp4,
+                SampleFile.html,
+            ].map { file in
+                Stub(
+                    response: Is(
+                        headers: [HTTPHeaders.contentType.rawValue : file.mimeType],
+                        body: file.body
+                    ),
+                    predicate: .equals(Request(path: "/\(file.rawValue)"))
+                )
+            },
+            recordRequests: true
+        )
+
+        let result = try await sut.postImposter(imposter: imposter)
+        print(result)
+    }
+
```

Comment:
@mat1th I've added this test so I could then go to localhost:1234 to check if all these content types go well. How can we best write a test for that?

Reply:
We do not have a test host (yet). So we can not test it with a snapshot. The only thing we can do now is validating that the response is correct;

Reply:
Yeah ok.